API/Configurations/OsuConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace API.Configurations
+{
+    public class OsuConfiguration
+    {
+        public const string Position = "Osu";
+
+        [Required(ErrorMessage = "ApiKey is required!")]
+        public string ApiKey { get; set; } = String.Empty;
```

Comment:
Throughout the codebase we tend to use `string.Empty` instead (adjust where used)